ecosystem/sep-0010.md

```diff
@@ -138,6 +142,7 @@ On success the endpoint must return `200 OK` HTTP status code and a JSON object
         * Add this operation if the server supports [Verifying the Client Domain](#verifying-the-client-domain) and the client provided a `client_domain` parameter in the request.
       * zero or more `manage_data(source: server account, ...)` reserved for future use
   * signature by the **Server Account**
+  * memo: derived from the `memo_type` and `memo` value passed by the **Client** if specified, omitted otherwise
```

Comment:
❗ Can you move this up to after line 129 so that it's inclusion in this list is similar to how we group that data fields in transactions in the XDR and in SDKs?